Makefile

```diff
@@ -51,5 +51,5 @@ push: build
 		if [[ "$(DOCKER_ORG)" = "ghcr.io/bufbuild" ]]; then \
 			$(DOCKER) pull $(DOCKER_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION} || exit 1; \
 		fi; \
-		$(BUF) alpha plugin push $${plugin_dir} $(BUF_PLUGIN_PUSH_ARGS) --image $(DOCKER_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION} || exit 1; \
+		$(BUF) beta registry plugin push $${plugin_dir} $(BUF_PLUGIN_PUSH_ARGS) --image $(DOCKER_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION} || exit 1; \
```

Comment:
Should we also update the version of buf in go.mod? I know we aren't calling this elsewhere in the CI builds any longer so don't need to update those.

Reply:
I think this uses the `buf` version that's currently installed, and internally we'll have this set to always be the latest. (I'm updating that next to make sure certain workflows continue to work).